buildSrc/src/main/java/com/r3/csde/BuildCPIsHelper.java

```diff
@@ -2,26 +2,33 @@
 
 import java.io.*;
 import java.util.LinkedList;
+import java.util.List;
 
+// todo: this class needs refactoring
```

Comment:
please create a JIRA and refer to it in the TODO comment ;)

Reply:
Fix other occurrences as well

Reply:
Done - see https://r3-cev.atlassian.net/browse/CORE-11624